### ISSUE TITLE
Training code missing - Request to add

### ISSUE THREAD

**Issue by bascurtiz** [flyingblackshark/jax-bs-roformer#1]
Great that u re-wrote it into JAX !
Since I have no experience with JAX, I'd love to see you add the training code too (training loop, loss calculation, etc.)

This could help with:
https://sites.research.google/trc/about/ for training compute
[TPU Research Cloud - About](https://sites.research.google/trc/about/)

See also:
https://jax.readthedocs.io/en/latest/notebooks/Neural_Network_and_Data_Loading.html

**Comment by flyingblackshark** [comment]
Added
